app/controllers/settings/people_controller.rb

```diff
@@ -17,7 +17,7 @@ def update
 
   def person_params
     params.require(:person).permit(:email, personable_attributes: [
-      :id, :first_name, :last_name, :password, :openai_key, :anthropic_key
+      :id, :first_name, :last_name, :password, :openai_key, :anthropic_key, preferences: [:color_theme]
```

Comment:
For consistency let’s name it `theme`. That’s what you used elsewhere (filename, etc) and it’s great to make things single word whenever possible.